ice.c

```diff
@@ -3183,6 +3227,78 @@ void *janus_ice_send_thread(void *data) {
 					}
 				}
 			}
+			if (stream && stream->do_transport_wide_cc) {
+				/* Create a transport wide feedback message*/
+				size_t size = 1300;
+				char rtcpbuf[1300];
+				
+				/* Lock session */
+				janus_mutex_lock(&handle->stream->mutex);
+				
+				/* Order packet list */
+				GSList* sorted = g_slist_sort(handle->stream->transport_wide_received_seq_nums,rtcp_transport_wide_cc_stats_comparator);
+				
+				/* Create full stats queue */
+				GQueue* packets = g_queue_new();
```

Comment:
Wouldn't it be better to just create the queue once, as a property of the stream, instead of allocating one everytime we need to send the feedback?